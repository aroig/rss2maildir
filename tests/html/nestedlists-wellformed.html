<dl>
    <dt>When the clock strikes one</dt>
    <dd>
        <ul>
            <li>bullet point 1</li>
            <li>bullet point 2</li>
        </ul>
    </dd>
    <dt>When the clock strikes two</dt>
    <dd>
        <ol>
            <li>first item</li>
            <li>second item</li>
        </ol>
    </dd>
</dl>
