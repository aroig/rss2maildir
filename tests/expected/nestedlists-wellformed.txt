When the clock strikes one::
     * bullet point 1
     * bullet point 2

When the clock strikes two::
      1. first item
      2. second item
